Source/abci/CMakeLists.txt

```diff
@@ -16,10 +16,17 @@ file(GLOB sources *.cpp *.h Foundation/*.h Foundation/*.cpp Importer/*.h Importe
 list(APPEND sources ${ISPC_SOURCES})
 list(APPEND sources ${ISPC_OBJECTS})
 
-add_plugin(abci
-    SOURCES ${sources}
-    PLUGINS_DIR ${CMAKE_INSTALL_PREFIX}/com.unity.formats.alembic/Runtime/Plugins/x86_64
+if (CMAKE_SYSTEM_PROCESSOR STREQUAL "ARM64")
```

Comment:
I am curious what will happen if building on a Silicon Mac? Will the Mac lib also end up in `Plugins/ARM64` folder?

Reply:
Very good point, that's is possibly correct.